exchanges/exchange.go

```diff
@@ -766,3 +812,76 @@ func (e *Base) GetAPIURLDefault() string {
 func (e *Base) GetAPIURLSecondaryDefault() string {
 	return e.APIUrlSecondaryDefault
 }
+
+// GetWithdrawPermissions will return each of the exchange's compatible withdrawal methods
+func (e *Base) GetWithdrawPermissions() string {
+	services := []string{}
+	for i := 0; i < 32; i++ {
+		var check uint32 = 1 << uint32(i)
+		if e.APIWithdrawPermissions&check != 0 {
+			switch check {
+			case AutoWithdrawCrypto:
+				services = append(services, AutoWithdrawCryptoText)
+				break
```

Comment:
I think these breaks might be unnecessary. Test code functionality without them. They should just break out of the switch at end of case anyway. Or I need another coffee.

Reply:
I was intending to break the for loop. From reading more, break will _break_ the most immediate breakable thing, which in this case is the switch statement. I'll rewrite it to get out of the loop properly